### PR TITLE
Milestone 3A - Holden Evers

### PR DESCRIPTION
**Pull request by HoldenEv**
## Developer: Holden Evers

Closes #140

### Pull Request Summary

created a rest api using MongoDB to all for the storage of the blog and project objects.

### Pull Request Checklist

- [X ] Code is neat, readable, and works
- [ X] Comments are appropriate
- [ X] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [X ] The developer name is specified
- [X ] The summary is completed
- [X ] Ask for a review in communication channels

### CODE REVIEW

**Review by jlaksana — CHANGES_REQUESTED** [review]
I think you forgot to push your milestone 3 code.

File: bootcamp-milestone-3A/src/database/projectSchema.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// import IBlog from "@/blogData";
```
Comment (ragdassi) [inline bootcamp-milestone-3A/src/database/projectSchema.ts:1]: looks good

File: bootcamp-milestone-3A/src/app/blog/page.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import Blog from "@/database/blogSchema";
+import Comment from "@/database/blogSchema";
+import styles from "./blog.module.css";
+import connectDB from "@/helpers/db";
+import BlogPreview from "../../../components/blogPreview";
+
+async function getBlogs() {
+  await connectDB();
```
Comment (ragdassi) [inline bootcamp-milestone-3A/src/app/blog/page.tsx:18]: nice job with the async function

File: bootcamp-milestone-3A/src/app/portfolio/page.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import Link from "next/link"
+import style from "./portfolio.module.css"; 
+import connectDB from "@/helpers/db";
+import Project from "@/database/projectSchema";
+import ProjectPreview from "../../../components/projectPreview";
+
+async function getProjects() {
```
Comment (ragdassi) [inline bootcamp-milestone-3A/src/app/portfolio/page.tsx:7]: nice work connecting your portfolio component with mongo db

**Comment by ragdassi** [comment]
MS 3a looks great!

**Pull request closed without merging**